packages/frontend/src/components/accounts/create/verify_account/VerifyOption.js

```diff
@@ -0,0 +1,246 @@
+    }
+
+    :before {
+        content: '';
+        height: 23px;
+        width: 23px;
+        top: 29px;
+        border: 2px solid #E6E6E6;
+        position: absolute;
+        border-radius: 50%;
+    }
+
+    .title {
+        color: #3F4045;
+        font-weight: 600;
+    }
+
+    .desc { 
+        color: #72727A;
+        max-width: 270px;
+        margin-top: 5px;
+
+        @media (max-width: 450px) {
+            max-width: 240px;
+        }
+
+        @media (max-width: 360px) {
+            max-width: 175px;
+        }
+    }
+
+    svg {
+        &.hardware-wallet-icon {
+            width: 66px;
+            margin-right: -23px;
+            position: absolute;
+            right: 0;
+            top: -6px;
+        }
+    }
+
+    &.active {
+        background-color: #F0F9FF;
+        cursor: default;
+        border-left: 4px solid #2B9AF4;
+        border-top-left-radius: 0;
+        border-bottom-left-radius: 0;
+
+        :before {
+            background-color: #2B9AF4;
+            border-color: #2B9AF4;
+        }
+
+        :after {
+            content: '';
+            position: absolute;
+            transform: rotate(45deg);
+            left: 21px;
+            top: 35px;
+            height: 11px;
+            width: 11px;
+            background-color: white;
+            border-radius: 50%;
+            box-shadow: 1px 0px 2px 0px #0000005;
+        }
+
+        .title {
+            color: #003560;
+        }
+
+        .desc {
+            color: #005497;
+        }
+
+        > hr {
+            border: 0;
+            background-color: #D6EDFF;
+            height: 1px;
+            margin: 20px -15px 0px -15px;
+        }
+    }
+    &.error {
+        border-color: #ff585d;
+    }
+
+    input {
+        margin-top: 20px;
+    }
+
+    .react-phone-number-input {
+        position: relative;
+
+        .react-phone-number-input__country {
+            position: absolute;
+            right: 0;
+            z-index: 1;
+            top: 50%;
+            transform: translateY(calc(-50% + 10px));
+        }
+
+        .react-phone-number-input__icon { 
+            &:not(.react-phone-number-input__icon--international) {
+                margin-right: 5px;
+            }
+        }
+
+        .react-phone-number-input__icon--international {
+            svg {
+                display: none;
+            }
+            
+            background-image: url(${IntFlagIcon});
+            background-repeat: no-repeat;
+        }
+
+        .react-phone-number-input__icon {
+            border: 0;
+        }
+
+        .react-phone-number-input__country-select-arrow {
+            width: 8px;
+            height: 8px;
+            border-color: black;
+            border-width: 0 1px 1px 0;
+            transform: rotate(45deg);
+            margin-top: -1px;
+            margin-left: 5px;
+            margin-right: 5px;
+        }
+
+        select {
+            font-size: 16px;
+        }
+    }
```

Comment:
Should these styles be in `VerifyWithEmailOption` since they don't apply _unless_ that component is being rendered?